src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -328,6 +337,20 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
                         isMinSelectableChoicesEnabled ? Integer.toString(minSelectableChoices) : "-1");
     }
 
+    /**
+     * Get the number of MSQ choices when ticking the option "students (excluding self)".
+     * Which is equals to the number of students - 1
+     * @param courseId the courseId
+     * @return number of MSQ choices
+     */
+    private int generateNumOfChoicesForStudentsExcludingSelf(String courseId) {
+
+        List<StudentAttributes> copyOfStudentList =
```

Comment:
This is not really a `copyOf`, is it?

Reply:
I meant it as a a copy of the student list(in the liberal meaning of copy i.e. a copy of this paper). Is there a specific meaning to copyOf? 
Anyway changed it to `studentList`.